samples/99_zil_interpreter/app/builtins.rb

```diff
@@ -262,3 +262,27 @@ def expect_minimum_argument_count!(arguments, min_count)
   false
 }
 
+ZIL_BUILTINS[:OBJECT] = lambda { |arguments, context|
```

Comment:
I think that it should eval its arguments, but that would need to be backed up by someone